hbase-server/src/main/java/org/apache/hadoop/hbase/master/RegionServerTracker.java

```diff
@@ -163,8 +168,9 @@ private synchronized void refresh() {
       server.abort("Unexpected zk exception getting RS nodes", e);
       return;
     }
-    Set<ServerName> servers =
+    Set<ServerName> servers = CollectionUtils.isEmpty(names) ? Collections.EMPTY_SET :
```

Comment:
Collections.emptySet(). Use EMPTY_SET will introduce a warning.

Reply:
OK, I will use replace it with `Collections.emptySet()`